features/liquidity/components/PoolCard.tsx

```diff
@@ -58,8 +62,8 @@ export const PoolCard = ({
     ? formatToCompactDollarValue(providedTotalLiquidity.dollarValue)
     : 0
 
-  const totalDollarValueLiquidityFormatted = dollarValueFormatterWithDecimals(
-    fluidLiquidity.total.dollarValue,
+  const totalDollarValueLiquidityFormatted = dollarValueFormatter(
+    parseInt(String(fluidLiquidity.total.dollarValue), 10),
```

Comment:
what does the 10 do?

Reply:
Second argument here sets out the math system. 10 in this case is for decimal one. I'm using the function to get rid of the decimal part. Could've probable just used `Math.ceil`